CorsixTH/Lua/languages/greek.lua

```diff
@@ -0,0 +1,3349 @@
+    financial_criteria_met = "Έχετε ικανοποιήσει τα οικονομικά κριτήρια για αυτό το επίπεδο. Τώρα κρατήστε τον προϋπολογισμό σας πάνω από %d, ενώ σιγουρεύετε ότι το νοσοκομείο σας λειτουργεί αποτελεσματικά",
+    dont_kill_more_patients = "Πραγματικά δεν έχετε το περιθώριο να πεθάνουν κι άλλοι ασθενείς!",
+    nearly_won = "Είστε πολύ κοντά στο να κερδίσετε αυτό το επίπεδο.",
+    nearly_lost = "Είστε πολύ κοντά στο να χάσετε αυτό το επίπεδο.",
+  },
+  staff_advice = {
+    need_doctors = "Χρειάζεστε περισσότερους Γιατρούς. Δοκιμάστε να τοποθετήσετε τους καλύτερους Γιατρούς σας στα δωμάτια με τις μεγαλύτερες ουρές.",
+    need_handyman_plants = "Θα χρειαστεί να προσλάβετε έναν Πολυτεχνίτη για να ποτίσει τα φυτά.",
+    need_handyman_litter = "Ο κόσμος έχει αρχίσει να ρυπαίνει με σκουπίδια το νοσοκομείο σας. Προσλάβετε έναν Πολυτεχνίτη για να τα καθαρίσει.",
+    too_many_doctors = "Έχετε πάρα πολλούς Γιατρούς. Κάποιοι από αυτούς δεν έχουν να απασχοληθούν με κάτι.",
+    too_many_nurses = "Νομίζω ότι αυτήν την στιγμή έχετε πάρα πολλές Νοσοκόμες.",
+    need_handyman_machines = "Θα χρειαστεί να προσλάβετε Πολυτεχνίτες αν θέλετε να συντηρήσετε τα μηχανήματα στο νοσοκομείο σας.",
+    need_nurses = "Θα χρειαστεί να προσλάβετε περισσότερες Νοσοκόμες. Οι Πτέρυγες και τα Φαρμακεία χρειάζονται μια Νοσοκόμα.",
+  },
+  multiplayer = {
+    players_failed = "Οι ακόλουθοι παίκτες απέτυχαν να ολοκληρώσουν τον τελευταίο στόχο : ",
+    objective_completed = "Έχετε ολοκληρώσει τον στόχο. Συγχαρητήρια!",
+    everyone_failed = "Κανείς δεν κατάφερε να ολοκληρώσει εκείνο τον τελευταίο στόχο. Οπότε όλοι συνεχίζουν το παιχνίδι !",
+    objective_failed = "Αποτύχατε να ολοκληρώσετε τον στόχο.",
+    poaching = {
+      in_progress = "Θα σας ενημερώσω σχετικά με το αν αυτό το άτομο επιθυμεί να έρθει να εργαστεί για εσάς.",
+      already_poached_by_someone = "Αποκλείεται! Κάποιος ήδη προσπαθεί να βουτήξει αυτό το άτομο.",
+      not_interested = "Χα! Δεν ενδιαφέρονται να εργαστούν για εσάς - είναι χαρούμενοι εκεί που είναι.",
+    },
+  },
+  vomit_wave = {
+    started = "Φαίνεται πως έχετε έναν ιό εμετού στο νοσοκομείο σας, αν το είχατε κρατήσει καθαρότερο αυτό δεν θα είχε συμβεί. Ίσως πρέπει να προσλάβετε περισσότερους Πολυτεχνίτες.",
+    ended = "Ουφ! Φαίνεται πως ο ιός που προκάλεσε εκείνον τον εμετό έχει λίγο-πολύ εξασθενήσει. Κρατήστε το νοσοκομείο σας καθαρό μελλοντικά.",
+  },
+}
+menu_options_volume = {
+  [80] = "  80%  ",
+  [50] = "  50%  ",
+  [100] = "  100%  ",
+  [70] = "  70%  ",
+  [40] = "  40%  ",
+  [90] = "  90%  ",
+  [60] = "  60%  ",
+  [20] = "  20%  ",
+  [30] = "  30%  ",
+  [10] = "  10%  ",
+}
+newspaper = {
+  [1] = {
+    [1] = "ΓΙΑΤΡΟΣ ΔΗΜΙΟΥΡΓΕΙ ΚΟΙΝΩΝΙΚΗ ΚΑΤΑΚΡΑΥΓΗ",
+    [2] = "ΤΡΕΛΟΣ ΓΙΑΤΡΟΣ ΠΑΡΙΣΤΑΝΕΙ ΤΟ ΘΕΟ",
+    [3] = "ΣΟΚ ΦΑΡΣΕΡ-ΣΤΑΪΝ",
+    [4] = "WHAT WAS ON LAB SLAB?",
+    [5] = "ΕΠΙΔΡΟΜΗ ΒΑΖΕΙ ΤΕΛΟΣ ΣΕ ΕΠΙΚΙΝΔΥΝΗ ΕΡΕΥΝΑ",
+  },
+  [2] = {
+    [1] = "DR-UNK AS A SKUNK",
```

Comment:
This one might've also been missed. But I suppose you could play with words to achieve a newsapaper title of similar effect.